bibliopixel/project/project.py

```diff
@@ -44,10 +45,44 @@ def make_animation(layout, animation, run=None):
     return animation
 
 
-def _make_project(path=None, animation=None, run=None, **kwds):
+def make_remote(layout, remote):
+    OFF_ANIM = {
+        'typename': 'bibliopixel.animation.off.OffAnim'
+    }
+
+    animations = remote.get('animations', None)
+    if not animations or len(animations) == 0:
+        raise ValueError('Must specify `animations` for remote.')
+
+    anim_list = []
+    for anim in animations:
+        run = anim.get('run', {})
+        run['threaded'] = True  # Remote anims must be threaded, maybe a little hacky?
```

Comment:
I think you might consider passing the threaded flag all the way down to the remote animation, and then having it report an error there, and keep going.

What happens here is if someone deliberately sets `threaded` false, you secretly ignore their (dumb) command.  You shouldn't secretly drop stuff on the floor!